frontend/app/src/composables/common.ts

```diff
@@ -71,8 +71,10 @@ export const useSectionLoading = () => {
 };
 
 export const isSectionLoading = (section: Section) => {
```

Comment:
@kelsos do we need to get rid of this too?

Reply:
If we can actually migrate it to use the other composable sure